application/src/main/java/net/furizon/backend/feature/pretix/objects/states/dto/CountryDataRaw.java

```diff
@@ -15,7 +18,11 @@ public class CountryDataRaw {
     @NotNull private final String phonePrefix;
 
     public CountryData toPhoneCountry() {
-        return new CountryData(name, code, phonePrefix);
+        var loc = Locale.of("", code);
+        Map<String, String> translatedDescription = Map.of(
+                Locale.ITALIAN.getLanguage(), loc.getDisplayCountry(Locale.ITALIAN),
```

Comment:
This MUST become a parametrized secondary language, or either a configurable list of supported languages